ui/components/ui/tab-empty-state/tab-empty-state.tsx

```diff
@@ -65,7 +65,7 @@ export const TabEmptyState: React.FC<TabEmptyStateProps> = ({
 }) => {
   return (
     <Box
-      className={twMerge(className, 'max-w-56')}
+      className={twMerge('max-w-56', className)}
```

Comment:
This fixes an issue with twMerge and overwriting classnames this should be noted for all components add the className prop last